src/test/java/com/ffxivcensus/gatherer/player/PlayerBuilderIT.java

```diff
@@ -61,6 +61,7 @@ public void testGetPlayer() throws Exception {
         assertTrue(playerOne.getLevelThaumaturge() >= 60);
         assertTrue(playerOne.getLevelArcanist() >= 60);
         assertTrue(playerOne.getLevelRedmage() >= 70);
+        assertTrue(playerOne.getLevelBluemage() >= 70);
```

Comment:
This is going to fail :) Suggest running `maven verify` so that you get all of the tests executed. (Note: Integration tests - those ending in `IT` and have external dependencies - don't get executed with `maven test`).